sessionctx/variable/varsutil.go

```diff
@@ -150,7 +150,7 @@ func GetSessionOnlySysVars(s *SessionVars, key string) (string, bool, error) {
 		if err != nil {
 			return "", false, err
 		}
-		return string(j), true, nil
+		return config.HideConfig(string(j)), true, nil
```

Comment:
ok i got to ask. why this can't be done by simply changing the `IndexUsageSyncLease` field to `json:"-"`

https://github.com/pingcap/tidb/blob/091ae3bde0a54b126f2f2ed275dd20b0986d74d3/config/config.go#L431